src/20240707_AaveV3Ethereum_IncreaseCCIPFacilitatorCapacity/IncreaseCCIPFacilitatorCapacity.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Increase CCIP Facilitator Capacity"
+author: "@karpatkey_TokenLogic"
+discussions: "https://governance.aave.com/t/arfc-gho-arb-increase-ccip-facilitator-capacity/18169"
+snapshot: "Direct-to-AIP"
+---
+
+## Simple Summary
+
+This publication proposes increasing the CCIP Facilitator Capacity on Arbitrum from 1M units to 2.5M units via the Direct-to-AIP process.
+
+## Motivation
+
+GHO was successfully launched on Arbitrum and the CCIP GHO Facilitator Bucket Capacity is 65% utilized. With $3.41M of liquidity and a solid peg during launch, there is sufficient depth to facilitate increasing the CCIP GHO Facilitator Bucket Capacity from 1M to 2.5M in line with the initial risk parameters proposed prior to launch 3.
+
+## Specification
+
+This proposal will increase the CCIP GHO Facilitator Bucket Capacity from 1M units to 2.5M units via the Direct-to-AIP process.
```

Comment:
I would also mention on the specification that we increase the bridgeLimit on ethereum as well

Reply:
good point, added.